modules/claims_api/app/workers/claims_api/ews_vbms_upload_job.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'claims_api/vbms_uploader'
+require 'claims_api/ews_vbms_sidekiq'
+
+module ClaimsApi
+  class EwsVBMSUploadJob
+    include Sidekiq::Worker
+    include ClaimsApi::EwsVBMSSidekiq
+
+    # Uploads an evidence waiver submission for a given claim to VBMS.
+    # If successfully uploaded, it queues a job to update the EWS code in BGS, as well.
+    #
+    # @param evidence_waiver_id [String] Unique identifier of the submitted EWS
+    def perform(evidence_waiver_id)
+      evidence_waiver = ClaimsApi::EvidenceWaiverSubmission.find(evidence_waiver_id)
+      uploader = ClaimsApi::EvidenceWaiverSubmissionUploader.new(evidence_waiver_id)
+      output_path = ClaimsApi::EvidenceWaiverBuilderJob.perform_async(evidence_waiver_id)
```

Comment:
hmmm.. this would have to be a `.new.perform` in this scenario or it will never succeed (I believe). I believe we'll get a job ID back. I also think we want the `EvidenceWaiverBuilderJob` to handle the update, similar to the `PoAFormBuilderJob`, then we could do a `.perform_async`.

Reply:
sorry, we probably mentioned `.perform_async`, but in this specific scenario is won't exist. I'm curious remove everything past the `BuilderJob` here and add the `upload_to_vbms` to the builder job.